pages/update.php

```diff
@@ -0,0 +1,97 @@
+<?php
+error_reporting(-1);
+ini_set('display_errors', 1);
+
+?>
+
+<?php
+require '../config/connect.php';
+require '../blocks/header.php';
+$client_id = ($_GET['id']);
+
+$sql = $pdo->prepare("SELECT * FROM client WHERE id = :client_id AND deleted_at IS NULL");
+$sql->execute(['client_id' => $client_id]);
+$client = $sql->fetch(PDO::FETCH_OBJ);
+
+
+if (!empty($client)){
+?>
+
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+    <link rel="icon" href="/img/favicon_gym.png">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css">
+    <link rel="stylesheet" href="/css/main.css">
+    <title></title>
+</head>
```

Comment:
It`s the same content which we have in header.php? Probably you need just switch body contend instead of switching full html